client/my-sites/site-settings/form-seo.jsx

```diff
@@ -320,12 +329,12 @@ export const SeoForm = React.createClass( {
 									prefix={ this.translate( 'Google' ) }
 									name="verification_code_google"
 									type="text"
-									value={ googleTag }
+									value={ googleCode }
 									id="verification_code_google"
 									disabled={ isDisabled }
 									isError={ hasError( 'google' ) }
 									placeholder={ getMetaTag( 'google', placeholderTagContent ) }
-									onChange={ event => this.handleVerificationCodeChange( event, 'google' ) } />
+									onChange={ event => this.handleVerificationCodeChange( event, 'googleCode' ) } />
```

Comment:
This is a minor thing and babel complains about it, but you're not supposed to use lambdas like this in jsx, because it means you are creating a new function each time the snippet gets rendered which degrades rendering performance. Here you're doing it to pass in the string "googleCode" into the verification function. But you could just read that from the event if you used "googleCode" as the input's name (and of course updated the formlabel accordingly). Then this would just be:
```js
onChange= { this.handlerVerificationCodeChange }
```